recipes-kernel/linux/files/patches-5.10/0218-arm64-dts-ti-iot2050-Refactor-the-m.2-and-minipcie-p.patch

```diff
@@ -33,7 +33,7 @@ index e73458ca6900..0dada0b39e36 100644
  		"", "", "", "", "", "", "", "", "", "",
  		"", "", "", "", "", "", "", "", "", "",
 diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
-index f473aedeb1f3..976b8eff45e0 100644
+index 94fdd776113f..871b86b3ff60 100644
```

Comment:
Seems this is noise?

Reply:
Not 100% sure but  I think this happened after realignment of the patch series. 
Both patches touching the "k3-am65-iot2050-common.dtsi" device tree and my change comes prior to the "noisy" patch which indeed affects the index line.